lib/datasource/pod/index.ts

```diff
@@ -32,13 +33,15 @@ function shardParts(lookupName: string): string[] {
 
 function releasesGithubUrl(
   lookupName: string,
-  opts: { account: string; repo: string; useShard: boolean }
+  opts: { hostURL: string; account: string; repo: string; useShard: boolean }
 ): string {
-  const { useShard, account, repo } = opts;
-  const prefix = 'https://api.github.com/repos';
+  const { hostURL, account, repo, useShard } = opts;
+  const prefix = hostURL
```

Comment:
Thanks, Kerry Kartchner for the suggestion: 

```suggestion
  const prefix = hostURL && !hostURL.includes('api.github.com')
```